apps/rush-lib/src/cli/scriptActions/BulkScriptAction.ts

```diff
@@ -184,11 +191,13 @@ export class BulkScriptAction extends BaseScriptAction {
     this.defineScriptParameters();
   }
 
-  private _mergeToProjects(): string[] {
-    const projects: string[] = [...this._toFlag.values];
-    if (this._toVersionPolicy.values && this._toVersionPolicy.values.length) {
+  private _mergeProjectsWithVersionPolicy(flags: CommandLineStringListParameter,
+    versionPolicies: CommandLineStringListParameter): string[] {
+
+    const projects: string[] = [...flags.values];
+    if (versionPolicies.values && versionPolicies.values.length) {
```

Comment:
Let's make this check more explicit.

```suggestion
    if (versionPolicies.values && versionPolicies.values.length > 0) {
```

Reply:
👍